server.go

```diff
@@ -88,6 +88,8 @@ func (s *Server) HandleReview(ctx context.Context, e *ReviewEvent) error {
 		return nil
 	}
 
+	s.poster.Status(ctx, e, PendingAnalysisStatus)
```

Comment:
maybe we should log error here?

Reply:
yes, I forgot about it. Fixed in 5c32b9f